lib/pbkit/pbkit.c

```diff
@@ -229,8 +229,11 @@ static  float           pb_BiasTable[7]={
                     1.907f,
                     2.0f    };
 
+static HAL_SHUTDOWN_REGISTRATION pb_shutdown_notification;
```

Comment:
nit: My original code was rather sloppy, I'd probably use `pb_shutdown_registration` now (pbkit is not sure about it's code style? `pb_ShutdownRegistration` is also an option).